{"name": "test_about", "status": "broken", "statusDetails": {"message": "AttributeError: 'async_generator' object has no attribute 'goto'", "trace": "page = <async_generator object page at 0x00000242BF91B850>\nenv_vars = {'password': 'User@123', 'url': 'https://worldamenities.com/', 'username': 'user123@gmail.com'}\n\n    @pytest.mark.smoke\n    @pytest.mark.asyncio\n    async def test_about(page, env_vars):\n        url = env_vars['url']\n    \n        login_page = LoginPage(page)\n>       await login_page.goto(url)\n\ntests\\test_WAWorld.py:24: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.LoginPage.LoginPage object at 0x00000242BF8F45C0>, WAurl = 'https://worldamenities.com/'\n\n    async def goto(self, WAurl: str):\n>       await self.page.goto(WAurl)\nE       AttributeError: 'async_generator' object has no attribute 'goto'\n\npages\\LoginPage.py:29: AttributeError"}, "start": 1720145346138, "stop": 1720145346138, "uuid": "06cf79dc-3d8f-4785-ac98-8e614a728cae", "historyId": "b63999fa594676011c2903251915f7b6", "testCaseId": "b63999fa594676011c2903251915f7b6", "fullName": "tests.test_WAWorld#test_about", "labels": [{"name": "tag", "value": "asyncio"}, {"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_WAWorld"}, {"name": "host", "value": "Feuji-V005L-R"}, {"name": "thread", "value": "5640-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_WAWorld"}]}